src/functions/process-code.func.ts

```diff
@@ -56,3 +58,10 @@ const findMapStyle = (rows: string[]) => {
 		mapStyle = mapStyle.toLocaleLowerCase().replace("style:", "").trim();
 	return mapStyle;
 };
+
+var findMapZoom = (rows) => {
```

Comment:
Please add type here so that the Typescript compiler now longer complains. Thanks!